### ISSUE TITLE
identity type usage

### ISSUE THREAD

**Issue by DanGrayson** [HoTT/book#1128]
Here is an example where an identity type is used in a sentence as though it were a proposition:

<img width="895" alt="Screen Shot 2022-10-13 at 8 52 08 AM" src="https://user-images.githubusercontent.com/700228/195645972-d4cbb01e-d66b-47fa-a314-48e981127aa4.png">

This is likely to cause confusion.

**Comment by mikeshulman** [comment]
Don't we do this all throughout mathematics?  "There are prime numbers p and q such that pq = 91."

**Comment by DanGrayson** [comment]
Isn't `pq=91` a proposition?

**Comment by awodey** [comment]
The correct form of elimination and computation rules for the higher constructors of HITs is a somewhat delicate matter - and still open to different approaches.  See section 6.2 for a discussion of some of the issues.  Note that in CTT the computation rule for loop is also definitional.

**Comment by mikeshulman** [comment]
The book doesn't reserve "proposition" to refer to only (-1)-types; it calls those "mere propositions".

**Comment by awodey** [comment]
MS: "The book doesn't reserve "proposition" to refer to only (-1)-types; it calls those "mere propositions"."
good point!
I think a compromise also determined that it's ok to simply display a (general) type, i.e. one that's not a mere proposition, and say that it "holds" (or similar) to mean that it's inhabited.

**Comment by DanGrayson** [comment]
> I think a compromise also determined that it's ok to simply display a (general) type, i.e. one that's not a mere proposition, and say that it "holds" (or similar) to mean that it's inhabited.

In the paragraph I highlighted above, you don't want to assert that the type is inhabited, you want to give an element of it.

**Comment by mikeshulman** [comment]
Giving an element of a type is the same as asserting that it is inhabited.  (Not to be confused with asserting that it is _merely_ inhabited!)

**Comment by DanGrayson** [comment]
Oops, you're right -- indeed, you say this: "when we say that A is inhabited, we mean that we have given a (particular) element of A, but that we are choosing not to give a name to that element" in 1.11.

**Comment by awodey** [comment]
true. But the convention (I guess) is that simply displaying a type such as a = b means the same as the judgement that the type is inhabited (or rather, the meta-statement that there is some t for which the judgement t : a = b holds).

**Comment by awodey** [comment]
that in response to MS: "There is a bit of lack of parallelism in these sentences of the form "such that P and Q" where P is a judgment and Q is a type, but I don't think that's very serious."

**Comment by mikeshulman** [comment]
Right, that's why I think it's not very serious.

**Comment by awodey** [comment]
agreed.

**Comment by DanGrayson** [comment]
If that's to be the convention, then this paragraph should probably be extended to explain it to the reader:

<img width="862" alt="Screen Shot 2022-10-13 at 3 36 46 PM" src="https://user-images.githubusercontent.com/700228/195723352-fbbec776-5d29-4ded-9dd7-0d311d4f1088.png">

**Comment by awodey** [comment]
Why isn't that just what that paragraph already explains?

**Comment by DanGrayson** [comment]
It should be explained that when you write "if X", or when you assert "X", you are regarding X as a proposition, in the way described.

**Comment by mikeshulman** [comment]
What else could we be doing?

**Comment by DanGrayson** [comment]
It won't be obvious to students.
